README.md

```diff
@@ -23,8 +23,8 @@ Builds can be found in the [Releases](https://github.com/revenge-mod/RevengeTwea
 
 ### Jailed
 
```

Comment:
I am not sure but the urls do not work (they are blank)

Reply:
they open for me

Reply:
https://github.com/user-attachments/assets/dd252453-8be2-45e7-a9fb-04fdeac7390f

Reply:
because they are not http links, they only work on iphones

Reply:
let me give you a vid

Reply:
Why is it a tinyurl link in the first place? This should be changed to a regular one. Images can also be uploaded directly in GitHub.

Reply:
https://files.catbox.moe/nvdmai.mov
file too big for github

Reply:
You can use https://streamable.com/ in the future but I get it.